packages/deployer/src/handlers/aws/index.ts

```diff
@@ -0,0 +1,54 @@
+import rawConfig from '../../config-data/config.json';
+import * as node from '@airnode/node';
+
+const config = node.config.parseConfig(rawConfig);
+
+function encodeBody(data: node.WorkerResponse): string {
+  return JSON.stringify(data);
+}
+
+export async function startCoordinator() {
+  await node.handlers.startCoordinator(config);
+  const response = { ok: true, data: { message: 'Coordinator completed' } };
+  return { statusCode: 200, body: encodeBody(response) };
+}
+
+export async function initializeProvider(event: any) {
+  // State and config are sent separately
+  const stateWithConfig = node.providerState.update(event.state, { config });
```

Comment:
Parameters were being read from a `parameters` field. That's not needed according to my debugging
https://github.com/api3dao/airnode/blob/f74e2a011aa56daed1c8f38c77c8fd90cea3ff93/packages/node/src/aws/handler.ts#L21

Reply:
Both of them right? 293bfd6